index.js

```diff
@@ -1,27 +1,50 @@
-let books = [];
+// class Books {
+//   constructor(books = []) {
+//     this.list = books;
+//   }
+// }
 
-const saveData = () => {
-  localStorage.setItem('myBooks', JSON.stringify(books));
-};
+// class Book {
+//   constructor(title, author, id) {
+//     this.title = title;
+//     this.author = author;
+//     this.id = id;
+//   }
+// }
 
-const addBook = (title, author) => {
-  const awesomeBook = {
-    title,
-    author,
-    id: books.length + 1,
+class Methods {
+  constructor(title, author, id) {
+    this.title = title;
+    this.author = author;
+    this.id = id;
+  }
+
+  addBook = (bookTitle, bookAuthor, bookId) => {
+    const book = {
+      title: bookTitle,
+      author: bookAuthor,
+      id: bookId,
+    };
+    this.books.push(book);
   };
-  books.push(awesomeBook);
-};
 
-const removeBook = (id) => {
-  books = books.filter((book) => book.id !== id);
+  removeBook = (id) => {
+    this.books.splice(this.books[id - 1], 1);
+  };
+}
+
+const methods = new Methods();
+methods.books = [];
```

Comment:
- [ ] Here you are adding a new `key` to your `Method` object, this should be done inside the class, for example, if your class holds a list of books, a good name could be `BookList` and the `constructor` could be something like this:
```javascript
constructor(title, author, id) {
  this.books = [ ];
}
```